aiida_pseudo/cli/install.py

```diff
@@ -23,43 +24,60 @@ def cmd_install():
 
 
 @cmd_install.command('family')
-@click.argument('archive', type=types.FileOrUrl(mode='rb'))
+@click.argument('archive_or_folder', type=types.PathOrUrl(exists=True, file_okay=True))
 @click.argument('label', type=click.STRING)
 @options_core.DESCRIPTION(help='Description for the family.')
 @options.ARCHIVE_FORMAT()
-@options.FAMILY_TYPE()
+@options.PSEUDO_TYPE()
```

Comment:
I'm not sure if we should remove `FAMILY_TYPE` input here. I was trying to install a family as a `CutoffsFamily`, since the basic `PseudoPotentialFamily` class doesn't have the `RecommendedCutoffMixin` as a parent class, and I wanted to add recommended cutoffs to be used with the `get_builder_from_protocol()` method.

Reply:
Also related to #78.